sklearn/metrics/tests/test_score_objects.py

```diff
@@ -499,25 +499,6 @@ def test_scorer_memmap_input():
         yield check_scorer_memmap, name
 
 
-def test_deprecated_names():
```

Comment:
I'm not sure if this should go out in version `0.20` and if so, if `classes.rst` is correct anymore. here:
https://github.com/scikit-learn/scikit-learn/blob/01e0639b15d515b446cb4589f01097b16fb4994a/doc/modules/classes.rst#L779-L805

Reply:
Yes, classes.rst is still valid. This deprecations only pertains to getting scorers by a string.